setcap.sh

```diff
@@ -12,7 +12,13 @@ if [ -x "$AVB_SETCAP_TOOL" ]; then
 	echo "Using setcap_tool defined on AVB_SETCAP_TOOL env var"
 	$AVB_SETCAP_TOOL $1 && exit
 	echo "Failed to set cap"
-elif [ `which setcap` ]; then
+fi
+if [ `which setcap_tool &> /dev/null ` ]; then
```

Comment:
I'm not sure if it's a good idea to change the `elif` for `if`. Supposed that first setcap_tool fails because current user doesn't have permission to do so. It will fail again up to more two times when trying again.